ctapipe/image/tests/test_cleaning.py

```diff
@@ -45,6 +46,58 @@ def test_dilate():
     assert dmask.sum() == 1 + 6 + 12 + 18
 
 
+def test_component():
+    from ctapipe.image import ImageCleaning, tailcuts_clean, fact_image_cleaning
+
+    geom = CameraGeometry.from_name('LSTCam')
+    image = np.random.poisson(12, geom.n_pixels)
+    arrival_times = np.random.uniform(0, 30, geom.n_pixels)
+
+    # test default is tailcuts with defaults
+    cleaning = ImageCleaning()
```

Comment:
Minor point: I think it would be nice to use a verb form for the instance name here, instead of a gerund, since the instance is supposed to feel like a simple function and we (try to) use verbs for functions...